python/src/file.rs

```diff
@@ -320,6 +320,11 @@ impl LanceFileWriter {
         self.inner_lock()?.add_schema_metadata(key, value);
         Ok(())
     }
+
+    pub fn abort(&self) -> PyResult<()> {
```

Comment:
We can call `abort` in `Drop` at the `RT.runtime`.

Reply:
Got it. Do you mean we should use Drop instead of an explicit abort method? It makes sense to me, let me fix it.